test/usb.coffee

```diff
@@ -172,7 +172,10 @@ describe 'Device', ->
             it 'polls the device', (done) ->
                 pkts = 0
 
-                inEndpoint.startPoll 8, 64
+                inEndpoint.startPoll 8, 64, (e, b, n) ->
+                    assert.ok(e == undefined, e)
```

Comment:
I don't think this will get called unless there is an error (in which case the assertion will fail!)

Should we be calling the callback after startPoll succeeds?

Reply:
That's a fair point, I was hyper fixated on my case where it was due to the error being emitted always, but yes, this should just call the callback during normal and errors cases, allowing the user to handle it.

Reply:
I believe the assertion that there is currently no error (which the test expects) is valid, but I fixed it so it will always call the callback upon emitting an end, with any error it may have as well.